packaging/skupper-router.spec.rpkg

```diff
@@ -63,19 +65,28 @@ BuildRequires: gcc-c++
 BuildRequires: cmake
 
 # skupper-router requirements
-BuildRequires: python3-devel
-BuildRequires: python3-setuptools
+%{?fedora:BuildRequires: python3-devel >= %{python_minimum_version}}
+%{?fedora:BuildRequires: python3-setuptools}
+%{?fedora:BuildRequires: python3-pip}
+# without wheel the installed files lack `python_qpid_proton-0.37.0.dist-info`
+%{?fedora:BuildRequires: python3-wheel}
+%{?rhel:BuildRequires: python39-devel >= %{python_minimum_version}}
+%{?rhel:BuildRequires: python39-setuptools}
+%{?rhel:BuildRequires: python39-pip}
+%{?rhel:BuildRequires: python39-wheel}
+%{?rhel:BuildRequires: python39-rpm-macros}
 BuildRequires: libwebsockets-devel >= %{libwebsockets_minimum_version}
 BuildRequires: libnghttp2-devel >= %{libnghttp2_minimum_version}
 # man pages --help
 BuildRequires: asciidoc
-BuildRequires: python3-qpid-proton >= %{proton_minimum_version}
 # check ctest
 BuildRequires: cyrus-sasl-plain
 
 # proton-c requirements
 BuildRequires: openssl-devel
 BuildRequires: cyrus-sasl-devel
+# python-qpid-proton requirements
+BuildRequires: swig
```

Comment:
yeah, this is now out of date, proton python is using cffi; but early this week it still worked with main